src/sign.ml

```diff
@@ -254,6 +255,14 @@ let read : string -> t = fun fname ->
     its dependencies. *)
 let add_rule : t -> sym -> rule -> unit = fun sign sym r ->
   sym.sym_rules := !(sym.sym_rules) @ [r];
+  (* EXPE *)
+  begin
+    let pama = Dtree.Pattmat.of_rules Timed.(!(sym.sym_rules)) in
```

Comment:
You can remove `Timed.` as this module is open.

Reply:
0cd9afe